.travis.yml

```diff
@@ -1,23 +1,27 @@
 # see http://about.travis-ci.org/docs/user/languages/php/ for more hints
 language: php
+
 php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
```

Comment:
The fast_finish option is only use to mark tests green without waiting
allowed_failure tests.

This change nothing on the tests.

Sullivan SENECHAL
Le 2 mai 2015 6:21 PM, "Ronan Guilloux" <notifications@github.com> a écrit :

> In .travis.yml
> <https://github.com/ronanguilloux/IsoCodes/pull/26#discussion_r29548583>:
>
> >  php:
> >    - 5.4
> >    - 5.5
> >    - 5.6
> > +  - 7.0
>
> fast_finish was blog-announced here
> <http://blog.travis-ci.com/2013-11-27-fast-finishing-builds/> and
> documented here](
> http://docs.travis-ci.com/user/build-configuration/#Fast-finishing).
>
> Only `PHP 7' <http://en.wikipedia.org/wiki/PHP#Release_history> is
> announced, and IMHO nightlyrelease support do not make sense for a
> business code library. So no need to add 7.1 so far.
>
> Since we're using more tiers, dev-related libraries, it is quite
> convenient to have all build processes details in only one go, in order to
> check what's going on: if any build will finish as soon as a job has
> failed, this wouldn't ease the debug process (see #20
> <https://github.com/ronanguilloux/IsoCodes/issues/20> and its related
> Travis output
> <https://travis-ci.org/ronanguilloux/IsoCodes/builds/59845133> by
> example).
>
> So I propose:
>
>    - no fast_finish so far
>    - no 7.1 CI tests so far
>
> @TomasVotruba <https://github.com/TomasVotruba>: Please consider the
> PHPUnit question below ;-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ronanguilloux/IsoCodes/pull/26/files#r29548583>.
>